bedrock/firefox/views.py

```diff
@@ -591,6 +591,33 @@ def get_template_names(self):
                 template = 'firefox/whatsnew/index.html'
         elif locale == 'id':
             template = 'firefox/whatsnew/index-lite.id.html'
+        elif version.startswith('78.'):
+            if variation == '2' and locale in ['en-US', 'en-CA', 'en-GB']:
+                template = 'firefox/whatsnew/whatsnew-fx78-a-en.html'
+            elif variation == '3' and locale in ['en-US', 'en-CA', 'en-GB']:
+                template = 'firefox/whatsnew/whatsnew-fx78-b-en.html'
+            elif variation == '4' and locale in ['en-US', 'en-CA', 'en-GB']:
+                template = 'firefox/whatsnew/whatsnew-fx78-c-en.html'
+            elif variation == '5' and locale in ['en-US', 'en-CA', 'en-GB']:
+                template = 'firefox/whatsnew/whatsnew-fx78-d-en.html'
+            elif variation == '2' and locale == 'de':
```

Comment:
When I set `Dev=False` in my `.env` and try to load this URL I get redirected to en-US because the locale isn't activated:

http://localhost:8000/de/firefox/78.0/whatsnew/all/?v=2

Adding the following to the templates seems like the simplest fix for the experiment variations:

```jinja
{% add_lang_files "firefox/whatsnew" %}
```